experiments/tester/src/RNTester/TestComponents/ThemeTest.tsx

```diff
@@ -31,6 +16,7 @@ const brandColors = {
   Outlook: ['#CCE3F5', '#B3D6F2', '#69AFE5', '#2488D8', '#0078D7', '#106EBE', '#1664A7', '#135995']
 };
 
+// This IProcessTheme takes the parent theme and shims in the brand colors selected in the RadioGroup
 const fakeBrandTheme: IThemeDefinition = (theme: ITheme): IPartialTheme => {
   if (brand === 'Office') return {};
```

Comment:
Do you have the Prettier plugin installed? It should have reformatted this code and wrapper the body inside a { } block.

Reply:
I'm pretty sure this is how prettier formatted this line of code.  I didn't author it that way.

Reply:
Hmm, maybe it just doesn't enforce if conditions to have { }. I'll adjust